static/openapi/faas-spec.json

```diff
@@ -0,0 +1,670 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "PhotonIQ FaaS API",
+    "description": "API documentation for the PhotonIQ FaaS Service",
+    "version": "1.0.0"
+  },
+  "servers": [
+    {
+      "url": "https://api-demo-faas.paas.macrometa.io",
+      "description": "PhotonIQ FaaS API"
+    }
+  ],
+  "paths": {
+    "/api/faas/v1/health": {
+      "get": {
+        "tags": [
+          "Health"
+        ],
+        "summary": "Check system health",
+        "description": "Check if system is up and running.",
+        "operationId": "handle_mm_health",
+        "responses": {
+          "200": {
+            "description": "The system is up and running."
+          }
+        }
+      }
+    },
+    "/api/faas/v1/deploy": {
+      "post": {
+        "tags": [
+          "Function"
+        ],
+        "summary": "Deploy the function",
+        "description": "Deploy the function for the specified key function name.",
```

Comment:
We will support other languages, this is why we are not writing WASMs

Reply:
AWS Lambda calls their functions "Lambda functions".
MS Azure calls them "Azure functions".
Google calls them "Cloud functions".

We could call them "FaaS functions".
Anyone have better ideas?

Reply:
I think not "FaaS Function" because in the end it is "Function as a Service Functions", it is repetitive and does not have identity.

Maybe "PhotonIQ Function" or "Macrometa Function"

Reply:
"Photoniq functions" it is, per Durga.

Reply:
I like it, but make sure the capitalization is "PhotonIQ."